web/src/app/projects/repository/repository.component.ts

```diff
@@ -27,11 +27,25 @@ export class RepositoryComponent implements OnInit {
   ngOnInit(): void {
     this.repositorySubscription = this.repositoryService
       .findOneById(this.uid)
-      .subscribe((repository: RepositoryModel) => this.repository = repository);
+      .subscribe((repository: RepositoryModel) => {
+        this.repository = repository;
+        this.sortMilestones();
+      });
   }
 
   ngDestroy(): void {
     this.repositorySubscription
       .unsubscribe();
   }
+
+  /**
+   * This function is used for sorting the milestones by updatedAt
+   */
+  sortMilestones(): void {
```

Comment:
Can this be moved to the milestone model? With the name like `getMilestonesByDate()`

Reply:
I am creating this function as generic so that it can be used to sort any list by date field in next PR.

Reply:
ok sounds good 👍 . In its own service would be good 🤓